versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/logic/RepositoryLogicImpl.java

```diff
@@ -244,7 +245,11 @@ private void initializeInternalRef(
       CreateCommit.Builder c =
           newCommitBuilder()
               .parentCommitId(EMPTY_OBJ_ID)
-              .headers(EMPTY_COMMIT_HEADERS)
+              .headers(
+                  CommitHeaders.newCommitHeaders()
+                      .add("repo.id", persist.config().repositoryId())
```

Comment:
It looks so, but it's not really.
Without this change, `AbstractBasePersistTests#scanAllObjects()` fails, because the commit objects between two repos become equal - indirectly caused by the necessary change to add `@Value.Auxiliary` to `CommitObj.created()`